Auth0/Authentication.swift

```diff
@@ -33,6 +33,94 @@ public typealias DatabaseUser = (email: String, username: String?, verified: Boo
+       .login(email: "support@auth0.com", code: "123456")
+       .start { result in
+           switch result {
+           case .success(let credentials):
+               print(credentials)
+           case .failure(let error):
+               print(error)
+           }
+       }
+    ```
+
+    You can also specify audience, scope, and additional parameters
+
+    ```
+    Auth0
+       .authentication(clientId: clientId, domain: "samples.auth0.com")
+       .login(email: "support@auth0.com",
+            code: "123456",
+            audience: "https://myapi.com/api",
+            scope: "openid email",
+            parameters: ["state": "a random state"])
+       .start { print($0) }
+    ```
+
+    When result is `Successful`, a `Credentials` object will be in it's associated value with at least an `access_token` (depending on the scopes used to authenticate)
+
+    - parameter email:             email the user used to start the passwordless login flow
+    - parameter code:              one time password (OTP) code the user received via email
+    - parameter audience:          API Identifier that the client is requesting access to. Default is `nil`
+    - parameter scope:             scope value requested when authenticating the user. Default is `openid`
+    - parameter parameters:        additional parameters that are optionally sent with the authentication request
+
+    - returns: authentication request that will yield Auth0 User Credentials
+    - seeAlso: Credentials
+    - requires: Passwordless OTP Grant `http://auth0.com/oauth/grant-type/passwordless/otp`. Check [our documentation](https://auth0.com/docs/clients/client-grant-types) for more info and how to enable it.
+    */
+    func login(email username: String, code otp: String, audience: String?, scope: String?, parameters: [String: Any]) -> Request<Credentials, AuthenticationError>
+    
+    /**
+    Logs in a user using a phone number and an OTP code received via sms (last part of the passwordless login flow)
+
+    ```
+    Auth0
+       .authentication(clientId: clientId, domain: "samples.auth0.com")
+       .login(phoneNumber: "+4599134762367", code: "123456")
+       .start { result in
+           switch result {
+           case .success(let credentials):
+               print(credentials)
+           case .failure(let error):
+               print(error)
+           }
+       }
+    ```
+
+    You can also specify audience, scope, and additional parameters
+
+    ```
+    Auth0
+       .authentication(clientId: clientId, domain: "samples.auth0.com")
+       .login(phoneNumber: "+4599134762367",
+            code: "123456",
+            audience: "https://myapi.com/api",
+            scope: "openid email",
+            parameters: ["state": "a random state"])
+       .start { print($0) }
+    ```
+
+    When result is `Successful`, a `Credentials` object will be in it's associated value with at least an `access_token` (depending on the scopes used to authenticate)
```

Comment:
```suggestion
    When result is `Successful`, a `Credentials` object will be in it's associated value with at least an `access_token` (depending on the scopes used to authenticate)
```

Sentence is a bit awkward, I'm not totally clear what it's trying to say.

Reply:
Fixed in b718c8441f66d8259087aec2166fbac980cef0ce - updated all instances of that sentence.